sql/updates/characters/2015_02_xx_xx_characters.sql

```diff
@@ -0,0 +1,95 @@
+DROP TABLE IF EXISTS `lag_reports`;
+
+ALTER TABLE `gm_surveys`
+    RENAME TO `gm_survey`,
```

Comment:
survey system will be done later. just adadpted all gm related tables to singular naming scheme.